runelite-client/src/test/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPluginTest.java

```diff
@@ -263,4 +263,61 @@ public void testTeleport()
 			menu("Cast", "Varrock Teleport", MenuAction.WIDGET_SECOND_OPTION),
 		}, argumentCaptor.getValue());
 	}
+
+	@Test
+	public void testTobDoor()
+	{
+		when(config.swapQuick() && config.swapHomePortal() != HouseMode.ENTER).thenReturn(true);
```

Comment:
You can't put expressions inside of when(), I'm surprised this works at all

Reply:
Changed the test a bit